server/controllers/DocumentControllers.js

```diff
@@ -0,0 +1,215 @@
+import pagination from '../helper/pagination';
+
+const Document = require('../models').Document;
+const User = require('../models').User;
+
+const DocumentControllers = {
+  /**
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} returns a response object.
+   */
+  create(req, res) {
+    const authorId = res.locals.decoded.UserId;
+    return Document.create({
+      title: req.body.title,
+      content: req.body.content,
+      UserId: authorId,
```

Comment:
Why is this PascalCase and `authorId` is camelCase. The standard is to use camelCase for properties or variables so change this to `userId` and keep consistent throughout the app